### PR TITLE
[MRG] Adding contributor's guidelines

### PR DESCRIPTION
**Pull request by NelleV**
This is still work in progress!

### CODE REVIEW

File: doc/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  the 'Fork' button near the top of the page. This creates a copy of the code
+  under your account on the GitHub user account. For more details on how to
+  fork a repository see [this
+  guide](https://help.github.com/articles/fork-a-repo/).
+
+- Clone your fork of the HiC-pro repo from your GitHub account to your local disk:
+
+```bash
+    $ git clone git@github.com:YourLogin/HiC-pro.git
+    $ cd scikit-learn
```
Comment (nservant) [inline doc/CONTRIBUTORS.md]: change by HiC-Pro ?

**Comment by NelleV** [comment]
Actually, scikit-learn has on the same page a diversity statement:

```
# Our community, our values

We are a community based on openness and friendly, didactic, discussions.

We aspire to treat everybody equally, and value their contributions.

Decisions are made based on technical merit and consensus.

Code is not the only way to help the project. Reviewing pull requests, answering questions to help others on mailing lists or issues, organizing and teaching tutorials, working on the website, improving the documentation, are all priceless contributions.

We abide by the principles of openness, respect, and consideration of others of the Python Software Foundation: https://www.python.org/psf/codeofconduct/
```

I really like it compared to other codes of conducts, but I'll let you see what you feel comfortable with.

**Comment by nservant** [comment]
That perfect. We should only change the last line ?
"of the Python Software Foundation: https://www.python.org/psf/codeofconduct/"
Could you add a paragraph on that ?
many thanks

**Comment by NelleV** [comment]
I've added a link to the UseR code of conduct as well as a diversity statement.

**Comment by NelleV** [comment]
We don't rely on HiTC?

**Comment by NelleV** [comment]
I think it's good to go!

**Comment by nservant** [comment]
Thanks !

**Pull request merged**